spec/models/period_filter_spec.rb

```diff
@@ -92,4 +92,24 @@
       end
     end
   end
+
+  describe '#legal_periods' do
+    subject(:legal_periods) do
+      described_class.new.legal_periods
+    end
+
+    context 'when NO percents exist for employee and employer' do
+      it 'returns the periods with the defaults added in' do
+        expect(legal_periods.first.employee_percent).to eq(1)
+        expect(legal_periods.first.employer_percent).to eq(1)
+      end
+    end
+
+    context 'when percents exist for employee and employer' do
+      it 'returns the periods with the defaults added in' do
+        expect(legal_periods[1].employee_percent).to eq(3)
```

Comment:
It seems that there are two different ways of accessing the element of the legal_periods array. See Line#103, 104 of  `spec/models/period_filter_spec.rb` and compare it with these two lines L#110, 111.